src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -23,31 +23,48 @@ monitors:
       sources:
         process:
           type: commandLine
-          commandLine: ps aux | grep '${var::processNames}'
+          commandLine: ps -eo user,pid,pcpu,pmem,etimes,command
           computes:
           - type: awk
             script: |
               {
+                user = $1
                 pid = $2
                 cpu = $3
                 mem = $4
-                command = $11
+                time = $5
+                cpu_time = (cpu / 100) * time
+                # Concatenate the full command from $6 to $NF
+                command = ""
+                for (i = 6; i <= NF; i++) {
+                  command = command $i " "
+                }
+                 # Remove the trailing space
+                command = substr(command, 1, length(command)-1)
 
-                # Converting time from HH:MM to seconds
-                time = $10
-                split(time, arr, ":")
-                time = arr[1] * 3600 + arr[2] * 60
-
-                printf "%s;%s;%s;%s;%s\n", pid, cpu, mem, time, command
+                printf "%s;%s;%s;%s;%s;%s;%s\n", user, pid, cpu, mem, time, cpu_time, command
               }
+          # Exclude the header row
+          - type: excludeMatchingLines
+            column: 1
+            regExp: '^USER'
+          # Filter by process command
```

Comment:
Add a filter on the process name with a 3rd variable ${var::processNameRegex}`

Actually, we should change the name of the variables as below:

* `matchName`: regular expression the process name (executable) must match with
* `matchCommand`: regular expression the process command must match with
* `matchUser`: regular expression the process user must match with